__tests__/schema/uuid/abstract-navigation-child.ts

```diff
@@ -281,200 +321,228 @@ describe('Page', () => {
             },
           },
         },
-      },
-      client,
-    })
+      })
   })
-})
 
```

Comment:
```suggestion
})
```
You (probably accidentally) reorganized the structure of the tests. The first three tests belong to 'Page' and the other three to 'Taxonomy Term'. The latter one should not be part of 'Page'.